noggin/controller/user.py

```diff
@@ -31,13 +31,15 @@ def user(ipa, username):
     # groups.
     member_groups = [
         Group(g)
-        for g in ipa.group_find(user=username, all=False, fasgroup=True)['result']
+        for g in ipa.group_find(
+            a_criteria=None, o_user=username, o_all=False, fasgroup=True
+        )['result']
     ]
     managed_groups = [
         Group(g)
-        for g in ipa.group_find(membermanager_user=username, all=False, fasgroup=True)[
-            'result'
-        ]
+        for g in ipa.group_find(
+            a_criteria=None, o_membermanager_user=username, o_all=False, fasgroup=True
```

Comment:
It's not necessary to specify `a_criteria=None` as it's the default.

Reply:
That makes sense.